geodiff/src/drivers/sqliteutils.cpp

```diff
@@ -199,6 +183,7 @@ std::string Sqlite3Value::toString( sqlite3_value *ppValue )
     val = "blob " + std::to_string( sqlite3_value_bytes( ppValue ) ) + " bytes";
   return val;
 }
+*/
```

Comment:
Do we need this commented block?

Reply:
I think it may be useful for debugging in future, so I was thinking the same.. I would keep it probably for now and then remove it if not needed